src/excel_templates/build.clj

```diff
@@ -1,10 +1,11 @@
 (ns excel-templates.build
-  (:import [java.io File FileOutputStream]
+  (:import [java.io File FileOutputStream FileInputStream]
            [java.util Calendar]
            [org.apache.poi.openxml4j.opc OPCPackage]
            [org.apache.poi.ss.usermodel Cell Row DateUtil WorkbookFactory]
            [org.apache.poi.xssf.streaming SXSSFWorkbook]
-           [org.apache.poi.xssf.usermodel XSSFWorkbook])
+           [org.apache.poi.xssf.usermodel XSSFWorkbook]
+           )
```

Comment:
Shaun, could you clean up these trailing parens? There are a bunch of them all down the file that should not be on a line by themselves.